TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/adapters/ReviewAdapter.java

```diff
@@ -0,0 +1,76 @@
+package com.fbu.thefoodienetwork.adapters;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.fbu.thefoodienetwork.databinding.ItemReviewBinding;
+import com.fbu.thefoodienetwork.models.ParseRestaurant;
+import com.fbu.thefoodienetwork.models.ParseReview;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+
+import java.util.List;
+
+public class ReviewAdapter extends RecyclerView.Adapter<ReviewAdapter.ViewHolder> {
+    private static final String TAG = "ReviewAdapter";
+    private Context context;
+    private List<ParseReview> reviewsList;
+
+    public ReviewAdapter(Context context, List<ParseReview> reviewsList) {
+        this.context = context;
+        this.reviewsList = reviewsList;
+    }
+
+    @NonNull
+    @Override
+    public ReviewAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        LayoutInflater layoutInflater = LayoutInflater.from(parent.getContext());
+        ItemReviewBinding binding = ItemReviewBinding.inflate(layoutInflater, parent, false);
+        return new ViewHolder(binding);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ReviewAdapter.ViewHolder holder, int position) {
+        ParseReview aReview = reviewsList.get(position);
```

Comment:
Just a heads up, usually when you prepend the letter "a" to a variable, it means that it's an abstract implementation.  In this case, a better name for this variable might be simply `parseReview`